cmd/configuration/configuration.go

```diff
@@ -77,12 +79,13 @@ func initFlytectlConfig(ctx context.Context, reader io.Reader) error {
 
 	templateValues := configutil.ConfigTemplateSpec{
 		Host:     "dns:///localhost:30081",
-		Insecure: initConfig.DefaultConfig.Insecure,
+		Insecure: true,
 	}
 	templateStr := configutil.GetSandboxTemplate()
 
 	if len(initConfig.DefaultConfig.Host) > 0 {
-		templateValues.Host = fmt.Sprintf("dns:///%v", initConfig.DefaultConfig.Host)
+		templateValues.Host = fmt.Sprintf("dns://%s", trim(initConfig.DefaultConfig.Host))
```

Comment:
After setting this can we validate that its valid dns endpoint

Reply:
we don't need that extra check

Reply:
It would be useful if user enters garbage data eg : https://0.0.0.0.0.0
Url.Parser might be able to catch it i suppose.